src/pybind/mgr/dashboard/controllers/mgr_modules.py

```diff
@@ -90,7 +111,13 @@ def _is_module_managed(self, module_name):
             this service, otherwise ``false``.
         :rtype: bool
         """
-        return module_name in self.managed_modules
+        if module_name in self.ignore_modules:
+            return False
+        mgr_map = mgr.get('mgr_map')
+        for module_config in mgr_map['available_modules']:
+            if module_name == module_config['name']:
+                return True
```

Comment:
Indeed, ``list()`` can be used here, but i'm in two minds because this method is doing much more than required. Because of that i tend to keep the current implementation.